analyzer/tests/functional/analyze_and_parse/test_files/simple1.deduplication.output

```diff
@@ -1,7 +1,7 @@
 NORMAL#CodeChecker log --output $LOGFILE$ --build "make deduplication" --quiet
-NORMAL#CodeChecker analyze $LOGFILE$ --output $OUTPUT$ --analyzers clangsa
+NORMAL#CodeChecker analyze $LOGFILE$ --output $OUTPUT$ --analyzers clangsa -j1
```

Comment:
In this file in the build command we build multiple translation units:
https://github.com/Ericsson/codechecker/blob/1617b22cedcc76f0d5bf6e2a3fcc5cd58d1ca56f/analyzer/tests/functional/analyze_and_parse/test_files/multi_error_skipped.output#L1
This can lead to a non-deterministic run.

To avoid non-deterministic test runs I recommend to explicitly specify the `-j1` option in every .output files under this directory. So if someone introduce a new test file and copies an existing file it will be hard to make such a mistake. What do you think @bruntib?

Reply:
I think adding `-j1` to each test file wouldn't force future test files to use it. So I implemented another solution instead, where we force `-j1` in test runner implicitly. What do you think about this?

Reply:
@bruntib You are right, it's a better solution :blush: Thanks!